addons/website/static/src/snippets/s_blockquote/001.scss

```diff
@@ -0,0 +1,58 @@
+.s_blockquote[data-vcss='001'] {
+    // Reset
+    border: 0;
+    padding: 0;
+    .s_blockquote_icon {
+        font-size: $font-size-base;
+    }
+    .blockquote-footer {
+        color: inherit;
+        .s_blockquote_author {
+            opacity: .75;
+        }
```

Comment:
That whole commit needs to be reviewed with the color system branch now being merged (wait for tomorrow, it is nearly all merged). Those two rules will become useless and since the `s_blockquote_author` is the only non-compatible change in the XML, I feel like you won't need a 001.scss file but directly improve the 000.scss to work with old and new XML.